src/group/managed_group/mod.rs

```diff
@@ -923,6 +926,26 @@ impl ManagedGroup {
     pub fn export_ratchet_tree(&self) -> Vec<Option<Node>> {
         self.group.tree().public_key_tree_copy()
     }
+
+    #[cfg(any(feature = "expose-test-vectors", test))]
+    pub fn export_path_secrets(&self) -> Vec<Secret> {
+        self.group.tree().private_tree().path_secrets().to_vec()
```

Comment:
Should we return the slice here? (not sure how it's used but a reference appears to be fine for the group context below)

Reply:
I think we run into the same problem as with the `credential` in the recent KeyStore PR, as the tree lives in a `RefCell`. See #387.

Reply:
Not really, you just have to call the `to_vec` on the outside if needed.

Reply:
Then I don't understand how to do it. I can't just omit the `to_vec()` and change the return type to `&[Secret]`. If I do that, `cargo` complains that I cannot return a value referencing a temporary value.

Reply:
Something like this would work. But also not too important...
```rust
pub fn export_path_secrets(&self) -> Ref<[Secret]> {
    Ref::map(self.group.tree(), |tree| tree.private_tree().path_secrets())
}
```

Reply:
Ah, I had totally forgotten about `Ref::map()`. I could probably use that for the `credential` case, too...